changelog.md

```diff
@@ -10,7 +10,8 @@
    * Under the Burning Suns
      * S08: Kromph and the Dust devil now trigger a critical event (issue #6211)
    * World Conquest
-     * Fixed artifacts "draining" HP on pick-up.  
+     * Fixed artifacts "draining" HP on pick-up.
+     * Fixed Bezoar artifact being bugged and broken.  
```

Comment:
Trailing whitespace on the new line, which was on the old line in the old version.

The reason I'm commenting on this is that @Wedge009 is doing a trailing-whitespace cleanup in #6498, which I think is going to merge first.

Reply:
what is this "trailing whitespace" you mention? How do I patch for it here?

Reply:
In your text editor, toggle visibility for non-printing characters. In your line 14 there's two additional spaces unnecessarily added after the full-stop - it would be best to remove that in your commit, please.

Reply:
Ah, I did the `Editor.renderWhitespace : "all",` option in `settings.json` and got to see 4 hidden whitespaces. Interesting.